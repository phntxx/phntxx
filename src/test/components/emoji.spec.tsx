import { render } from "@testing-library/react";
import Emoji, { IEmojiProps } from "../../components/emoji";

describe("emoji.tsx", () => {
  it("should render component without label without crashing", () => {
    const { asFragment } = render(<Emoji symbol="ðŸ¤”" />);
    expect(asFragment).toMatchSnapshot();
  });

  it("should render component with label without crashing", () => {
    const { asFragment } = render(<Emoji symbol="ðŸ¤”" label="Test" />);
    expect(asFragment).toMatchSnapshot();
  });
});
