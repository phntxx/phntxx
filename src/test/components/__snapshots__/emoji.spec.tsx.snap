// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`emoji.tsx should render component with label without crashing 1`] = `[Function]`;

exports[`emoji.tsx should render component without label without crashing 1`] = `[Function]`;
